### PR TITLE
Fix typo and exclude node_modules

### PR DESCRIPTION
**Pull request by sandervdo**
Noticed a small typo in the readme file.

Next to that I noticed that after following the instructions, node_modules was not ignored. Added this folder to the gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,5 @@
 /bin/
 /composer.phar
 /.idea
-/.vagrant
\ No newline at end of file
+/.vagrant
+node_modules
```
Comment (ChristianVermeulen) [inline .gitignore]: /node_modules? :)

Comment (sandervdo) [inline .gitignore]: Not necessary, this excludes the folder and it's contents. No need to prepend with a slash. :)

**Comment by ChristianVermeulen** [comment]
Just resolve conflicts and I will merge :)

**Comment by sandervdo** [comment]
Consider it done,

**Pull request merged**